internal/cli/atlas/deployments/options/deployment_opts.go

```diff
@@ -222,9 +228,12 @@ func (opts *DeploymentOpts) PromptDeploymentType() error {
 		return err
 	}
 
-	if !strings.EqualFold(opts.DeploymentType, AtlasCluster) && !strings.EqualFold(opts.DeploymentType, LocalCluster) {
-		return fmt.Errorf("%w: %s", errDeploymentTypeNotImplemented, deploymentTypeDescription[opts.DeploymentType])
-	}
+	return ValidateDeploymentType(opts.DeploymentType)
```

Comment:
[q] is this validation needed given we are using a select?

Reply:
great catch, there's no need to validate there, but looking at how this PromptDeploymentType func was used I realized we had few small validation bugs in few other commands: we were prompting and validating the type only if it was not provided as a flag. But the commands was accepting any value for the --type flag.
I fixed this in last commit